src/Model/Shopgate/Order.php

```diff
@@ -109,4 +107,33 @@ public function save()
     {
         $this->_getResource()->save($this);
     }
+
+    /**
+     * Get all shipments for the order
+     *
+     * @return array
+     */
+    public function getReportedShippingCollections()
+    {
+        $data = $this->getData('reported_shipping_collections');
+        $data = unserialize($data);
+        if (!$data) {
+            $data = [];
+        }
+
+        return $data;
+    }
+
+    /**
+     * @param array $collectionIds
```

Comment:
maybe a bit more strict like `string[]` or `int[]` for clarity of expectations

Reply:
done